src/sprints.md

```diff
@@ -46,3 +48,57 @@ sprints.
+
+- [PyLadies](https://github.com/pyladies) - PyLadies is a global mentorship group focused on helping more marginalized genders become active participants & leads in the Python open-source community.
+
+### Web Development Projects
+
+- [Django](https://github.com/django/django) - The Web framework for perfectionists with deadlines.
+
+- [Django Debug Toolbar](https://github.com/django-commons/django-debug-toolbar/issues) - Django Debug Toolbar is a configurable set of panels that display various debug information about the current request/response.
+
+- [BeeWare](https://github.com/beeware/beeware.github.io/issues) - BeeWare makes it easy to build and distribute GUI applications using pure Python. It includes frontend, mobile and packaging.
+
+- [Render Engine](https://github.com/render-engine/render-engine/issues) - Render Engine is a Python Static Site Generator built with flexibility prioritized.
+
+### Data Science / Scientific
+
+- [pandas](https://github.com/pandas-dev/pandas) - Pandas is a flexible and powerful data analysis / manipulation library for Python, providing labeled data structures similar to R data.frame objects, statistical functions, and much more.
+
+- [NumPy](https://github.com/numpy/numpy) - NumPy is a fundamental package for scientific computing with Python.
+
+- [SymPy](https://github.com/sympy/sympy/issues) - SymPy is a computer algebra system written in pure Python.
+
+- [Spyder](https://github.com/spyder-ide/spyder/issues) - SPYDER(Scientific Python Development Environment) is an open-source cross-platform integrated development environment (IDE) for scientific programming in the Python language.
+
+- [SciPy](https://github.com/scipy/scipy/issues) - SciPy is a free and open-source Python library used for scientific and technical computing. It provides utility functions for optimization, stats and signal processing.
+
+### Language / Core
+
+- [CPython](https://github.com/python/cpython/issues/) - The Python programming language.
+
+### Libraries
+
+- [Pillow](https://github.com/python-pillow/Pillow) - Pillow is the friendly Python Imaging Library(PIL) fork that allows you to manipulate images and perform basic image processing tasks.
+
+### Sustainability
+
+- **Sustainability Projects** (University of Manchester). The key project will be the **[Research Software Quality Kit][RSQKIT]** RSQKit is - part of the [EVERSE][EVERSE] project - which collects best practices for improving research software.  We'll focus on Green Software Sustainability content & tools, including recent work on [Green MetaData][SSIBLOG_GREENMD].
+
+The sprint leader is [Michael Sparks][MS_LINK] ( [University Of Manchester][UNIMAN] , [Physics][PHYSICS] & [eScience Lab][ESCIENCE], exBBC R&D ). He created the original BBC micro:bit prototype (which was distributed to schools around the UK), founded the Kamaelia project, helped co-found PyconUK in 2007, and is passionate about mentoring, on-boarding, and knowledge sharing.
```

Comment:
Can we indent this paragraph so it's clear it's about the leader for the bullet point above, and not the whole Sustainability category? (in the unlikely event that we have another one for this category).

Sadly I think prefixing this line with the appropriate number of `&nbsp;&nbsp;` might be the easiest way to do that

Reply:
FWIW, looking at the screenshot above I agree with @rebkwok - for the same reason! :-)

Reply:
On it